### PR TITLE
Bump LTS versions + ember-cli-dependency-checker versions

### PR DESCRIPTION
**Pull request by thoov**
Working on CI failures with floating deps.

Bumping LTS version to 3.16

### CODE REVIEW

File: packages/addon-template/package.json [hunk 1]
```diff
@@ -37,7 +37,7 @@
     "babel-eslint": "^10.1.0",
     "broccoli-asset-rev": "^3.0.0",
     "ember-cli": "~4.0.1",
-    "ember-cli-dependency-checker": "^3.2.0",
+    "ember-cli-dependency-checker": "3.2.0",
```
Comment (rwjblue) [inline packages/addon-template/package.json:40]:
```suggestion
    "ember-cli-dependency-checker": "^3.3.1",
```

Comment (rwjblue) [reply packages/addon-template/package.json:40]: The fix was landed to cli-dependency-checker

**Comment by thoov** [comment]
@rwjblue / @ef4 could you review

**Comment by ef4** [comment]
I think you're getting trolled by changes in either scenario-tester or Volta that are breaking our [per-scenario node version control](https://github.com/ef4/ember-auto-import/blob/44bd54db1ddf876ee542c73af651a1dadc395d4e/test-scenarios/scenarios.ts#L8-L17).

I think that is the source of the failures. We're supposed to be using old node to test old ember.

I would very much prefer to not do a semver major just to drop ember versions that are actually working fine.

**Comment by ef4** [comment]
Attempting to fix in https://github.com/ef4/ember-auto-import/pull/517

**Comment by ef4** [comment]
Superseded by #517.

**Pull request closed without merging**